src/EnergyPlus/FaultsManager.cc

```diff
@@ -1442,6 +1442,11 @@ namespace FaultsManager {
         // read faults input of Fault_type 109: Fouled Air Filters
         for (int jFault_AirFilter = 1; jFault_AirFilter <= NumFaultyAirFilter; ++jFault_AirFilter) {
 
+            // Read in chiller if not done yet
+            if (state.fans.GetFanInputFlag) {
+                Fans::GetFanInput(state.fans);
+            }
+
```

Comment:
Perhaps we should call GetFanInput here, similar to what's done a few lines above:

https://github.com/NREL/EnergyPlus/blob/3d2145e57854707476a667f73c514417625964aa/src/EnergyPlus/FaultsManager.cc#L1325-L1330

Not sure, can be reverted easily.

Reply:
I prefer the GetFanInputFlag() is called right at the point of use with Fault AirFilter as fixed. Need to correct the comment "Read in chiller if not done yet".